Mathlib/Algebra/GroupPower/CovariantClass.lean

```diff
@@ -175,19 +175,18 @@ variable [Preorder β] [CovariantClass M M (· * ·) (· ≤ ·)]
   [CovariantClass M M (swap (· * ·)) (· ≤ ·)]
 
 @[to_additive Monotone.const_nsmul]
-theorem Monotone.pow_right {f : β → M} (hf : Monotone f) : ∀ n : ℕ, Monotone fun a => f a ^ n
+theorem Monotone.pow_const {f : β → M} (hf : Monotone f) : ∀ n : ℕ, Monotone fun a => f a ^ n
   | 0 => by simpa using monotone_const
   | n + 1 => by
     simp_rw [pow_succ]
-    exact hf.mul' (Monotone.pow_right hf _)
-#align monotone.pow_right Monotone.pow_right
+    exact hf.mul' (Monotone.pow_const hf _)
+#align monotone.pow_right Monotone.pow_const
```

Comment:
These don't look like deprecations to me; can you split them to their own PR?

Reply:
Those lemma renames were announced in #9095. I just messed up when doing so.